client/structs/structs.go

```diff
@@ -184,3 +187,87 @@ func (d *DriverNetwork) Hash() []byte {
 	}
 	return h.Sum(nil)
 }
+
+type FingerprintRequest struct {
+	Config *config.Config
+	Node   *structs.Node
+}
+
+type FingerprintResponse struct {
+	attributes map[string]string
```

Comment:
These should be public.

Reply:
There are getters and setters for these, to prevent against uninitialized fields. Alternatively, we could have a constructor for FingerprintResponse which would do initialization of values all at once.